docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md

```diff
@@ -0,0 +1,125 @@
+# Dasharo Compatibility: Logo customization functionality
+
+## Test cases common documentation
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../../generic-test-setup/#firmware).
+1. Proceed with the
+    [Generic test setup: OS installer](../../generic-test-setup/#os-installer).
+1. Proceed with the
+    [Generic test setup: OS installation](../../generic-test-setup/#os-installation).
+1. Proceed with the
+    [Generic test setup: OS boot from disk](../../generic-test-setup/#os-boot-from-disk).
+
+## LCM001.001 Replace logo in existing image and flashing firmware
+
+**Test description**
+
+This test aims to verify whether it's possible to replace logo in existing
+firmware image and flash this firmware on the DUT.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. `OPERATING_SYSTEM` = Ubuntu 22.04
+
+**Test setup**
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into the system by using the proper login and password.
+1. Proceed with the
+    [Replace logo in an existing image](../../common-coreboot-docs/custom_logo.md#replace-logo-in-an-existing-image)
+    section.
+1. Reboot DUT.
+
+**Expected result**
+
+1. While booting custom logo appears on screen.
+
+## LCM002.001 Build image with custom logo and flashing firmware
+
+**Test description**
+
+This test aims to verify whether it's possible to build firmware image from the
+source and flash it on the DUT.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. `OPERATING_SYSTEM` = Ubuntu 22.04
+
+**Test setup**
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into the system by using the proper login and password.
+1. Proceed with the
```

Comment:
At this point, we should explain briefly, what operations should be performed. Linking to the documentation without mentioning, what part of it should be used might confuse the tester.

Reply:
Whole section should be performed